components/content/inspira/ui/text-highlight/TextHighlight.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <span :class="cn('inline-block px-1 pb-1', props.class)"><slot /></span>
+</template>
+
+<script setup lang="ts">
+import type { HTMLAttributes } from "vue";
+import { cn } from "@/lib/utils";
+
+interface Props {
+  delay?: number;
+  duration?: number;
+  class?: HTMLAttributes["class"];
+  textEndColor?: string;
+}
+
+const props = withDefaults(defineProps<Props>(), {
+  delay: 0,
+  duration: 2000,
+  class: undefined,
```

Comment:
No need to explicitly set the class `undefined`, it is undefined by default

Reply:
Maybe we should set up ESLint on the repo ?
https://eslint.vuejs.org/rules/require-default-prop.html (and disable this rules if we don't want it 😅 )

Removing `class: undefined` for this PR anyway.